sdk/examples/how_tos/accounts_and_addresses/list_addresses.rs

```diff
@@ -3,17 +3,24 @@
 
 //! In this example we will list all addresses of an account.
 //!
-//! `cargo run --release --all-features --example list_addresses`
+//! ```sh
+//! cargo run --release --all-features --example list_addresses
+//! ```
+
+use std::env::var;
 
 use iota_sdk::wallet::{Result, Wallet};
 
 #[tokio::main]
 async fn main() -> Result<()> {
-    // Create the wallet
-    let wallet = Wallet::builder().finish().await?;
+    // This example uses secrets in environment variables for simplicity which should not be done in production.
+    dotenvy::dotenv().ok();
 
-    // Get the account
-    let account = wallet.get_account("Alice").await?;
+    let wallet = Wallet::builder()
+        .with_storage_path(&var("WALLET_DB_PATH").unwrap())
+        .finish()
+        .await?;
+    let account = wallet.get_account(&var("ACCOUNT_ALIAS_1").unwrap()).await?;
```

Comment:
I think it's better like this.

Reply:
I think it's definitely worse. We have to understand this are just just random examples, these are the how tos that will be shown on the website for new users to learn. Things like that make it hard to process. But I guess I can live with it.

Reply:
After some thought I think you're probably right. For tests it makes sense to do this, but not so much for examples.

Reply:
I am not sure ... a dev that sees like 50 examples (or more if we take all binding examples into account) and still doesn't feel like running away might be confident enough to decipher it!? :trollface: 

Jokes aside ... I don't really see an alternative. I felt similar which is why I at least imported the `var` function. I wouldn't enforce "Alice" as an account name, and even if we do I would create a let binding because sometimes use the alias in a println.

Reply:
An alternative could be to just get the first account by index `wallet.get_account(0).await?` and not care about any aliases

Reply:
I think that's for the best. We have to consider examples as reading material, not necessarily perfect code.

Reply:
done